Tools/Scripts/filter-build-webkit

```diff
@@ -23,6 +23,20 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Filters the output of build-webkit into a more human-readable format.
+
+# To develop:
+#   make debug VERBOSITY=noisy >build-output.txt 
+#   cat build-output.txt | Tools/Scripts/filter-build-webkit --format=debug > formatted.txt
+
```

Comment:
I was confused for a while about whether the following was a continuation of "To develop" comment, or a separate one. Seems like it's the former.

Perhaps adding a "#" to this line would serve as a gentle hint to that effect?